debian/changelog

```diff
@@ -1,3 +1,7 @@
+  [Yen-Shin Chen]
+  * Enable to get the patch vertices by indicate the specific fault patch.
```

Comment:
Sorry Yen-Shin, I do not understand the English here. What does it mean *Enable to get the patch vertices by indicate the specific fault patch*? Can you rephrase that in a clearer sentence?